### PR TITLE
803 edit course unique individuals

### PR DESCRIPTION
**Pull request by abidingotter**
Please forgive the auto-formatting of the imports...

### CODE REVIEW

File: Source/Plugins/Core/com.equella.core/js/tsrc/course/EditCourse.tsx [hunk 1]
```diff
@@ -1,50 +1,29 @@
...
-  MenuItem,
-  Paper,
-  Switch,
-  Tab,
-  Tabs,
-  TextField,
-  Theme
-} from "@material-ui/core";
-import { sprintf } from "sprintf-js";
+import { Button, FormControl, FormControlLabel, FormGroup, FormHelperText, Grid, Input, InputLabel, MenuItem, Paper, Switch, Tab, Tabs, TextField, Theme } from "@material-ui/core";
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/course/EditCourse.tsx]: this line is getting a tab bit long, could it be formatting with one destructured variable per line?

File: Source/Plugins/Core/com.equella.core/js/tsrc/course/EditCourse.tsx [hunk 2]
```diff
@@ -1,50 +1,29 @@
...
+import { Bridge } from "../api/bridge";
 import { Error, Loader } from "../components/index";
 import MessageInfo from "../components/MessageInfo";
-import {
-  EditEntityDispatchProps,
-  EditEntityProps,
-  EditEntityStateProps,
-  entityStrings
-} from "../entity";
+import { EditEntityDispatchProps, EditEntityProps, EditEntityStateProps, entityStrings } from "../entity";
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/course/EditCourse.tsx]: same as above, this line is getting a tad long.

File: Source/Plugins/Core/com.equella.core/js/tsrc/course/EditCourse.tsx [hunk 3]
```diff
@@ -197,12 +176,13 @@ class EditCourse extends React.Component<Props, EditCourseState> {
 
   handleSave = () => {
     if (this.props.entity) {
-      const { versionSelection } = this.props.entity;
+      const { versionSelection, students } = this.props.entity;
       const vs = versionSelection === "DEFAULT" ? undefined : versionSelection;
-
+      const st = (students === 0 ? undefined : students);
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/tsrc/course/EditCourse.tsx]: Are there cases where the difference between `false`, `0`, `undefined`, and `null` would be significant here?
If not coud
```suggestion
      const st = students || undefined;
```
offer a more concise alternative?

Comment (abidingotter) [reply Source/Plugins/Core/com.equella.core/js/tsrc/course/EditCourse.tsx]: Sounds good

File: Source/Plugins/Core/com.equella.core/js/tsrc/course/EditCourse.tsx (hunk 1 above)
Comment (doolse) [reply Source/Plugins/Core/com.equella.core/js/tsrc/course/EditCourse.tsx]: @abidingotter do you not have the pre-commit hooks setup?

Comment (abidingotter) [reply Source/Plugins/Core/com.equella.core/js/tsrc/course/EditCourse.tsx]: Hmmm, I did... not sure what happened.  Have installed in Prettier in VS code now anyway.

**Pull request merged**